packages/calcite-components/src/components/text-area/assets/t9n/messages.json

```diff
@@ -1,4 +1,5 @@
 {
   "invalid": "Invalid",
-  "tooLong": "The current character length is {currentLength}, which exceeds the maximum character length of {maxLength}."
+  "tooLong": "The current character length is {currentLength}, which exceeds the maximum character length of {maxLength}.",
+  "tooShort": "The current character length is {currentLength}, which is less than the minimum character length of {minLength}."
```

Comment:
@geospatialem could you review the string wording?

Reply:
Awesome, @Elijbet - wording LGTM! 🚀